x/emissions/genesis.go

```diff
@@ -10,15 +10,17 @@ import (
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, ak types.AccountKeeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
+
+	// This create module accounts if not created
+	// TODO: find a proper way  to create module accounts?
```

Comment:
We can add methods for withdrawable emissions . That would need a change in proto files though

Reply:
Yes, does it have a logical on-chain usage right now? It seems right now it is only used for documentation purposes.
But it should be straightforward to add.

Maybe we could try to maintain somewhere a TODO list for when adding new state components so we always add the value in the genesis in the same PR we add the type.